README.md

```diff
@@ -89,6 +90,12 @@ To run this app locally, follow these steps:
    ```console
    pixi run -e frontend flutter-run
    ```
+   
+   Alternatively, you can explicitly set the `SUPABASE_URL` and `SUPABASE_ANON_KEY` values with the command below to ensure the application connects properly to Supabase
```

Comment:
This is not alternatively, but rather a required thing for developing the app. You can simply update line 91. Also, could you mention where the values can be retrieved from?